scripts/proxy_launchd.sh

```
@@ -1,10 +1,9 @@
 #!/bin/bash
-# Launchd-friendly proxy runner that keeps the uvicorn process in the foreground
 set -euo pipefail
 
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-VENV_PATH="$SCRIPT_DIR/venv"
-DEFAULT_RUNTIME_DIR="${CODEX_PROXY_RUNTIME_DIR:-/tmp/codex_plus}"
+SCRIPT_DIR="/Users/jleechan/projects_other/codex_plus"
```

Shellcheck (0.11.0)

[warning] SCRIPT_DIR appears unused. Verify use (or export if used externally).

(SC2034)